conftest.py

```diff
@@ -89,25 +82,13 @@ def pytest_runtest_setup(item):
     """
     Use incremental
     """
-    BASIC_LOGGER.info(f"\n{separator(symbol_='-', val=item.name)}")
     set_up_pytest_runtest_phase(item=item, phase="SETUP")
 
 
 def pytest_sessionfinish(session, exitstatus):
     reporter = session.config.pluginmanager.get_plugin("terminalreporter")
     deselected_str = "deselected"
-    deselected = len(reporter.stats.get(deselected_str, []))
-    summary = (
```

Comment:
this should not e removed

Reply:
Why? this is for pytest console/log
Should not be use the `get_logger()`, should keep using `BASIC_LOGGER`

Reply:
done